Project1/Task10-homework/Collections1.cs

```diff
@@ -0,0 +1,243 @@
+            var myListF = new List<int>() { 4, 3, 7, 8, 6 };
+            var myListG = new List<string>() { "aka", "maka", "saka", "lola", "kola" };
+            SortMerge(myListF, myListG);
+
+            //            Create a City class where there are fields int population, double area.Create a dictionary where Key is the name of the city and Value is the corresponding name of the cityand the object of type City. Create 5 elements for the dictionary. 
+            //Sort the dictionary by city area and display it on your console
+            //Browse the dictionary by population in reverse order and display it on your console
+            //Count the total population of all cities and output to the console *
+            Dictionary<string, City> myCity = new Dictionary<string, City>();
+            myCity.Add("Oku", new City(23, 456.5));
+            myCity.Add("Aka", new City(34, 345.4));
+            myCity.Add("mika", new City(24, 355.2));
+            myCity.Add("Lala", new City(44, 445.4));
+            myCity.Add("Saka", new City(54, 555.2));
+            var sorted = myCity.OrderBy(x => x.Value.Area);
+            Console.WriteLine("sorted dictionary by area:");
+            foreach (var item in sorted)
+            {
+                Console.WriteLine(item.ToString());
+            }
+            var reversed = myCity.OrderByDescending(x => x.Value.Population);
+            Console.WriteLine("decending order by population");
+            foreach (var item in reversed)
+            {
+                Console.WriteLine(item.ToString());
+            }
+            double sum = myCity.Sum(v => v.Value.Area);
+            Console.WriteLine($"Sum of area:{sum}");
+        }
+        public static void SortMerge(List<int> i, List<string> s)
+        {
+            i.Sort();
+            s.Sort();
+            s.Reverse();
+            var dic = new Dictionary<int, string>();
+            foreach (var key in i)
+            {
+                foreach (var value in s)
+                {
+                    dic[key] = value;
+                    s.Remove(value);
+                    break;
+
+                }
+            }
+            foreach (var item2 in dic)
+            {
+                Console.WriteLine(item2);
+            }
+        }
+
+        public static void reverseQueue(Queue<int> q)
+        {
+            if (q.Count == 0)
+                return;
+            int fr = (int)q.Peek();
+            q.Dequeue();
+            reverseQueue(q);
+            q.Enqueue(fr);
+        }
+
+        public static void GetMaxValue(Queue<int> q)
+        {
+            Console.WriteLine("Max element:" + q.Peek());
+        }
+
+        public static void DeleteMaxValue(Queue<int> q)
```

Comment:
Are you sure? If you name the method like `DeleteMaxValue` so it should look for the max value and delete it, but here I see only Dequeue call

Reply:
if I call Dequeue it just gonna delete the first element of Queue